queries.sql

```diff
@@ -1,10 +1,112 @@
 /*Queries that provide answers to the questions from all projects.*/
+SELECT
+  *
+FROM
+  animals
+WHERE
+  name LIKE '%mon';
 
-SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
-SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
-SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered = TRUE;
-SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE round(weight_kg::numeric, 2) >= 10.23 AND round(weight_kg::numeric, 2) <= 17.3;
+SELECT
+  name
+FROM
+  animals
```

Comment:
- I can see that you forgot to add the transactions mentioned in the project requirements, So please add the following transactions:
  - [ ] inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
  - [ ] Inside a transaction:
  Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
  Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
  Commit the transaction.
  Verify that change was made and persists after commit.
  - [ ] Inside a transaction delete all records in the animals table, then roll back the transaction.
  - [ ] Inside a transaction:
  Delete all animals born after Jan 1st, 2022.
  Create a savepoint for the transaction.
  Update all animals' weight to be their weight multiplied by -1.
  Rollback to the savepoint
  Update all animals' weights that are negative to be their weight multiplied by -1.
  Commit transaction

![image](https://user-images.githubusercontent.com/68796232/183631000-a122ec7e-6f09-4f59-a425-ef81bf7578a5.png)

Reply:
> * I can see that you forgot to add the transactions mentioned in the project requirements, So please add the following transactions:
>   
>   * [ ]  inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
>   * [ ]  Inside a transaction:
>     Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
>     Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
>     Commit the transaction.
>     Verify that change was made and persists after commit.
>   * [ ]  Inside a transaction delete all records in the animals table, then roll back the transaction.
>   * [ ]  Inside a transaction:
>     Delete all animals born after Jan 1st, 2022.
>     Create a savepoint for the transaction.
>     Update all animals' weight to be their weight multiplied by -1.
>     Rollback to the savepoint
>     Update all animals' weights that are negative to be their weight multiplied by -1.
>     Commit transaction
> 
> ![image](https://user-images.githubusercontent.com/68796232/183631000-a122ec7e-6f09-4f59-a425-ef81bf7578a5.png)

Hello @Omar-Muhamad, I added the transaction results as screenshots in the PR, since I didn't see a requirement saying that we should add it to some files!

Reply:
Hi @teckim, I know that you added screenshots with these transactions but Please add these transactions to the queries.sql file too.